src/main/java/gift/doamin/wishlist/service/WishListService.java

```diff
@@ -0,0 +1,69 @@
+package gift.doamin.wishlist.service;
+
+import gift.doamin.wishlist.dto.WishListForm;
+import gift.doamin.wishlist.entity.WishList;
+import gift.doamin.wishlist.repository.WishListRepository;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+public class WishListService {
+
+    private final WishListRepository wishListRepository;
+
+    public WishListService(WishListRepository wishListRepository) {
+        this.wishListRepository = wishListRepository;
+    }
+
+    public WishList create(Long userId, WishListForm wishListForm) {
+        Long productId = wishListForm.getProductId();
+        if (wishListRepository.existsByUserIdAndProductId(userId, productId)) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "동일한 상품을 위시리스트에 또 넣을수는 없습니다");
+        }
+        // 수량 0개는 등록 불가
+        if (wishListForm.getQuantity().equals(0)) {
```

Comment:
이 부분이 getter 사용을 지양해야하는 부분입니다. 좋은 예시네요.
```suggestion
        if (wishListForm.isZeroQuantity()) {
```
이런식으로 구현하는게 객체지향적인 방식입니다.